source/installation/packages.rst

```diff
@@ -21,12 +21,17 @@ can help with simple config questions, for anything else we concentrate on
 current versions.
 
 The rsyslog project offers current packages for a number of major distributions.
-More information about these can be found at http://www.rsyslog.com/downloads/download-other/.
+More information about these can be found at the |RsyslogPackageDownloads|_
+page.
 
 If you do not find a suitable package for your distribution, there is no
-reason to panic. You can use the :doc:`rsyslog docker containers <rsyslog_docker>`
-instead or
-:doc:`install rsyslog from the source tarball <install_from_source>`.
```

Comment:
same comment

Reply:
Here I moved the links to the "see also" section. This was as much an effort to pull out the specific documents and highlight them (via a "see also" section) as much as it was to allow the actual page titles to show through. I personally like the approach of using "see also" sections to bring attention to specific documents as I feel it allows for easy/quick browsing of a doc in order to find related items.

I don't know. I have *my* preferences, but ultimately this is up to you and I understand that.

Reply:
I am not totally sure, but it sounds reasonable. This reminds me we have a task to make the generation result available, which, as you said, is helpful for such discussions. Let's go ahead with the current proposal.